docs/clients/dotnet/21.2/migration-to-gRPC.md

```diff
@@ -625,6 +625,248 @@ You should be careful in defining the retry policy. Not all operations are by de
 
 ## Subscriptions
 TODO
+## Catch-up Subscriptions
+
+We unified catch-up subscriptions API in the gRPC client. In TCP client, you have multiple methods for subscribing to EventStoreDB, e.g. `SubscribeToStreamAsync`, `SubscribeToStreamFrom`, `FilteredSubscribeToAllAsync` etc. Now you have two main options:
+- subscribing to a single stream using `SubscribeToStreamAsync`,
+- subscribing to the `$all` stream using `SubscribeToAllAsync`.
+
+Both methods include overloads for specific configurations and optional parameters with sane defaults. The recommendation is to use the default settings and modify them when needed.
+
+### Positions
+
+As for other operations, we aligned subscriptions checkpoints/positions handling. Accordingly, as for [reading](#reading-events) instead of using nullable long value, we introduced classes:
+- `StreamPosition` for a single stream subscription,
+- `Position` for the `$all` stream.
+
+Each subscription method has an overload taking either a specific position or the default one (from the start). For instance, if you were previously using a unified approach:
+
+```csharp
+long? streamPosition = GetLastCheckpoint();
+
+tcpConnection.SubscribeToStreamFrom(
+    streamName,
+    streamPosition,
+    false,
+    EventAppeared,
+    SubscriptionDropped
+);
+```
+
+Now you'd need to call different overloads:
+
+```csharp
+StreamPosition? streamPosition = GetLastCheckpoint();
+
+if (streamPosition.HasValue)
+{
+    await grpcClient.SubscribeToStreamAsync(
+        streamName,
+        streamPosition.Value,
+        EventAppeared,
+        false,
+        SubscriptionDropped
+    );
+}
+
+await grpcClient.SubscribeToStreamAsync(
```

Comment:
I personally try to avoid `else,` but I agree that it's more explicit and more precise in this case. Updated in: https://github.com/EventStore/documentation/pull/498/commits/533a42f79130bf695ab4657b0c00b21570b2c11c.